.github/ISSUE_TEMPLATE/add-development-request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Add Development Request
```

Comment:
I think this all looks fine, but I'm curious if there is a way to preview how this will actually look?

Reply:
@JessicaMeixner-NOAA On the top right, click on **...** and select **view file**

Reply:
![Screen Shot 2021-03-17 at 9 34 03 PM](https://user-images.githubusercontent.com/22549231/111560254-8012db00-8768-11eb-9652-7f6672820659.png)
So this is what this will look like?

Reply:
I think so. As you suggested, we can merge and modify if further changes are required.